BUILD.bazel

```diff
@@ -1,13 +1,13 @@
 # Load the cc_library rule.
 load("@rules_cc//cc:defs.bzl", "cc_library")
-
 load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
 
 expand_template(
-    name = "catch_user_config.hpp",
+    name = "catch_user_config",
     out = "catch2/catch_user_config.hpp",
     substitutions = {
         "#cmakedefine CATCH_CONFIG_ANDROID_LOGWRITE": "",
+        "#cmakedefine CATCH_CONFIG_BAZEL_SUPPORT": " #define CATCH_CONFIG_BAZEL_SUPPORT",
```

Comment:
Is there a reason for the space before `#define`?

Reply:
Well spotted, just a typo, will fix, thanks!